src/huak/config/pyproject/toml.rs

```diff
@@ -173,6 +241,29 @@ build-backend = "huak.core.build.api"
 
         let toml = Toml::from(string).unwrap();
 
-        assert!(toml.project.authors.iter().nth(1).is_some());
+        assert_eq!(
+            "Troy Kohler",
```

Comment:
Is this a random name? Or is this leftover from some docs that were used?

Reply:
No clue where the name came from, but it was definitely there beforehand in the toml string for that test.